source/Nevermore/Advanced/SourceQueryBuilder.cs

```diff
@@ -71,7 +71,8 @@ public class SubquerySourceBuilder<TRecord> : SourceQueryBuilder<TRecord>, ISubq
         readonly ISelect select;
         string alias;
 
-        public SubquerySourceBuilder(ISelect select,
+        public SubquerySourceBuilder(ISelect select, 
+            string alias,
```

Comment:
This is a tad confusing - this adds the alias as a constructor param, when there's already a builder method to set the alias. If you removed this param, that'd ditch the need to have a `null` [here](https://github.com/OctopusDeploy/Nevermore/pull/107/files?file-filters%5B%5D=.cs#diff-259ccf05853011d388caa6554900d568R183)